packages/interface/src/GraphQLLanguageService.js

```diff
@@ -62,10 +62,12 @@ import {getASTNodeAtPosition} from 'graphql-language-service-utils';
 export class GraphQLLanguageService {
   _graphQLCache: GraphQLCache;
   _graphQLConfig: GraphQLConfig;
+  _extensions: ?Array<any>;
 
-  constructor(cache: GraphQLCache) {
+  constructor(cache: GraphQLCache, extensions: ?Array<any>) {
```

Comment:
It's not `any` it should be a function that accepts `GraphQLConfig` and return `GraphQLConfig`